### PR TITLE
Add footer extra links template

### LINKED ISSUE #9575: Add Custom Footer Links
## Description

Currently additional links can only be added to `extra_links.tmpl`. However, it adds links to head navbar. Some pages (e.g. Privacy Policy) are customarily linked in the footer.

I will shortly create a PR for this issue.

### PR DESCRIPTION
**Pull request by bagasme**
This PR will add `custom/extra_links_footer` template for adding additional links to footer.

Docs coming soon...

This will close #9575.

Please note that there isn't border separator between Gitea version and additional links. This caused by `&.:first-child` from https://github.com/go-gitea/gitea/blob/master/web_src/less/_base.less#L789-L791. I can't figure out how to resolve this.

### CODE REVIEW

**Review by 6543 — APPROVED** [review]

File: templates/base/footer_content.tmpl [hunk 1]
```diff
@@ -2,6 +2,9 @@
 	<div class="ui container">
 		<div class="ui left">
 			© Gitea {{if (or .ShowFooterVersion .PageIsAdmin)}}{{.i18n.Tr "version"}}: {{AppVer}}{{end}} {{if ShowFooterTemplateLoadTime}}{{.i18n.Tr "page"}}: <strong>{{LoadTimes .PageStartTime}}</strong> {{.i18n.Tr "template"}}: <strong>{{call .TmplLoadTimes}}</strong>{{end}}
+			<span class="ui right links">
+				{{template "custom/extra_links_footer" .}}
```
Comment (lafriks) [inline templates/base/footer_content.tmpl]: I think this should be moved between line 9 and 10 below and span is not needed anymore than

Comment (bagasme) [reply templates/base/footer_content.tmpl]: @lafriks I encapsulate the template inside `span` so that links can be put beside gitea version

Comment (lafriks) [reply templates/base/footer_content.tmpl]: Imho better would be to add them between home page link and go version string

File: docs/content/doc/advanced/customizing-gitea.en-us.md [hunk 2]
```diff
@@ -80,10 +80,10 @@ Dont forget to restart your gitea to apply the changes.
 
 ### Adding links and tabs
 
-If all you want is to add extra links to the top navigation bar, or extra tabs to the repository view, you can put them in `extra_links.tmpl` and `extra_tabs.tmpl` inside your `custom/templates/custom/` directory.
+If all you want is to add extra links to the top navigation bar, or extra tabs to the repository view, you can put them in `extra_links.tmpl` (links added to the navbar), `extra_links_footer.tmpl` (links added to the left side of footer), and `extra_tabs.tmpl` inside your `custom/templates/custom/` directory.
```
Comment (davidsvantesson) [inline docs/content/doc/advanced/customizing-gitea.en-us.md]:
```suggestion
If all you want is to add extra links to the top navigation bar or footer, or extra tabs to the repository view, you can put them in `extra_links.tmpl` (links added to the navbar), `extra_links_footer.tmpl` (links added to the left side of footer), and `extra_tabs.tmpl` inside your `custom/templates/custom/` directory.
````

**Review by davidsvantesson — APPROVED** [review]

**Review by lafriks — APPROVED** [review]

**Comment by lafriks** [comment]
Make LG-TM wotk

**Pull request merged**